ray_on_golem/server/models.py

```diff
@@ -14,6 +14,12 @@ class NodeState(Enum):
     stopping = "stopping"
 
 
+class ShutdownState(Enum):
```

Comment:
maybe not now, but if, in the future, we want to support various transitions of the server/cluster state, this might benefit from being a state machine...